defaults/main.yml

```diff
@@ -1,12 +1,39 @@
 ---
 postgresql:
-  data_directory: "/var/lib/postgresql/9.5/main"
-  locale: "en_GB.UTF-8"
-  max_connections: 100
-  port: 5432
-  shared_buffers: 128MB
+  listen_addresse: 127.0.0.1
+  locale: en_GB.UTF-8
+  root_password: P4ssw0rd!123
+  root_user: root
   ssl: true
-  timezone: "UTC"
   version: 9.5
-  root_user: root
-  root_password: P4ssw0rd!123
+
+  config:
+    data_directory: /var/lib/postgresql/9.5/main
+    # change to * to bind to 0.0.0.0
+    listen_addresses: 127.0.0.1
+    max_connections: 100
+    port: 5432
+    # defaults to 20%
+    shared_buffers: "{{ (ansible_memtotal_mb / 5) | int }}"
+    timezone: UTC
+
+  # put any extra config that is not currently
+  config_extra: { }
```

Comment:
it's a simple way for adding additional simple config to "postgresql.conf" without the need of changing this role (open for extension close for modification).

```
roles:
  - role: sansible.postgresql
    postgresql:
      config_extra:
        log_destination: syslog
```